packages/fast-app-shell-react/src/app-shell/app-shell.spec.tsx

```diff
@@ -0,0 +1,5 @@
+describe("AppShell", (): void => {
+    test("temp", (): void => {
+        expect(2).toBe(2);
```

Comment:
should we leave this placeholder here or at least just test to make sure the component doesn't throw?